### ISSUE TITLE
why choose --minqual 1 --mincov 2?

### ISSUE THREAD

**Issue by Tonny-zhou** [VishnuRaghuram94/AgrVATE#13]
Thank you for this great tool to detecting the mutations in agr operon!

I noticed that the values for --minqual  and --mincov are 1 and 2, respectively. So why you choose these values?

Best regards,
Tonny_z

**Comment by VishnuRaghuram94** [comment]
Hi Tonny,

Thank you for using AgrVATE! 

From some initial testing I found that snippy was "clipping" the first and last few basepairs of the extracted _agr_ operon before starting variant calling, this led to missing some variants in the operon. Since AgrVATE works with fasta assemblies, I am assuming the read quality and coverage checks have already been performed so I don't need snippy to perform them again. Therefore I set the minqual and mincov to minimum values so that all the "pseudoreads" snippy generates from the input fasta assembly are counted towards variant calling. 

I hope this helps,
Best,
Vishnu
